app/src/main/res/values/strings.xml

```diff
@@ -535,4 +535,8 @@
     <string name="tickets_for">Ticket(s) for %1$s</string>
     <string name="error_making_paypal_payment_message">Fail on making Paypal payment</string>
 
+    <!-- Non-Translatable -->
+    <string name="facebookHeadUrl" translatable="false">https://www.facebook.com/</string>
+    <string name="twitterHeadUrl" translatable="false">https://www.twitter.com/</string>
+    <string name="instagramHeadUrl" translatable="false">https://www.instagram.com/</string>
```

Comment:
snake case

Reply:
oh.. I thought we used Camel case.. I will fix this